### PR TITLE
LPS-174328 Move functional API tests to Echo and Lima

### PR DESCRIPTION
**Pull request by david-gutierrez-mesa**
https://issues.liferay.com/browse/LPS-174328

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/headless/headless-admin-content/headless-admin-content-test/src/testFunctional/tests/ExposeStructuredContentFolderIdInHeadlessAdminContent.testcase [hunk 1]
```diff
@@ -4,7 +4,7 @@ definition {
 
 	property portal.release = "true";
 	property portal.upstream = "true";
-	property testray.main.component.name = "Headless";
+	property testray.main.component.name = "Echo Headless";
```
Comment (magjed4289) [inline modules/apps/headless/headless-admin-content/headless-admin-content-test/src/testFunctional/tests/ExposeStructuredContentFolderIdInHeadlessAdminContent.testcase:7]: I just realised, I think we need to change the **@component-name = "portal-headless"** to the echo and lima ones as well

**Review by magjed4289 — CHANGES_REQUESTED** [review]

**Comment by david-gutierrez-mesa** [comment]
@austinchiang and @magjed4289 can you please review it?

File: modules/apps/headless/headless-admin-content/headless-admin-content-test/src/testFunctional/tests/ExposeStructuredContentFolderIdInHeadlessAdminContent.testcase (hunk 1 above)
Comment (david-gutierrez-mesa) [reply modules/apps/headless/headless-admin-content/headless-admin-content-test/src/testFunctional/tests/ExposeStructuredContentFolderIdInHeadlessAdminContent.testcase:7]: done! PR has been updated.

**Review by magjed4289 — APPROVED** [review]

**Pull request closed without merging**